test/linalg/symmetric.jl

```diff
@@ -250,3 +250,36 @@ let a = randn(2,2)
     cc = copy(c)
     @test conj!(c) == conj(Array(c))
 end
+
+# 19225
+let X = sparse([1 -1; -1 1])
```

Comment:
I removed the `sparse` wrapper for these tests, then copied a simplified version of what's currently here to the sparse tests. Let me know if the format looks okay.